adapters/adhese/adhese.go

```diff
@@ -0,0 +1,247 @@
+package adhese
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"sort"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type AdheseAdapter struct {
+	http             *adapters.HTTPAdapter
+	endpointTemplate template.Template
+}
+
+func (a *AdheseAdapter) Name() string {
```

Comment:
This is legacy as well and is not needed. You only need to implement the methods of `adapters.Bidder`. I'll try to find time in the next couple of weeks to update our new Adapter docs to make this clear. Sorry for misleading you on this.